apps/core/views.py

```diff
@@ -7,12 +7,21 @@
 import json
 from geopy.geocoders import MapBox
 import os
+import pytz
```

Comment:
Where did you use this???

Reply:
It's part of the datetime stuff. not 100% sure if its being used. django recommends installing it https://docs.djangoproject.com/en/3.0/topics/i18n/timezones/

Reply:
removed. will test locally

Reply:
seems to be needed and branch is working correctly. going to merge.